share/spice/libraries/triggers.txt

```diff
@@ -0,0 +1,11 @@
+go
```

Comment:
> API doesn't support "go get" queries.

Yes, sorry about the confusion. I meant we should add additional triggers to capture more queries, but add more logic to map those to appropriate platforms, to pass the correct name to the API. I've left a comment above that should clarify.